exercises/practice/hello-world/bin/fetch-cobolcheck

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-# This file is a copy of the
-# https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet file.
-# Please submit bugfixes/improvements to the above file to ensure that all tracks benefit from the changes.
+# This file is inspired by https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet.
+# It is only used in the cobol track, and a copy of it is placed in every exercise folder.
+# If you change something, make sure to upgrade all scripts in all exercises!
```

Comment:
@iHiD, Is this change sufficient to close #29

Reply:
Yep. Thanks :)